theano/tensor/signal/downsample.py

```diff
@@ -101,6 +101,10 @@ def out_shape(imgshape, ds, ignore_border=False, st=None):
             extra row/col of partial downsampling (False) or ignore it (True).
         :type ignore_border: bool
 
+        :param padding: (pad_h, pad_w), pad zeros on four borders
+            of the images, pad_h for padding rows, and pad_w for columns.
```

Comment:
That is a bit ambiguous: "padding rows" could mean that you pad each row so the rows are longer, but actually it adds new rows on top and bottom. Can you say that `pad_h` is the size of the top and bottom borders, and `pad_w` is the size of the left and right bordres instead?